src/core/Form/Textarea/Textarea.tsx

```diff
@@ -98,6 +106,7 @@ class BaseTextarea extends Component<TextareaProps> {
 
     return (
       <HtmlDiv
+        {...containerProps}
```

Comment:
Is className in the main props? I assume it is applied below, but now also containerProps includes a classname. This needs to be applied below or omitted from props in my opinion.

Reply:
It should be already be omitted from the TextareaProps's `containerProps`:

`containerProps?: Omit<HtmlDivProps, 'className'>;`

Reply:
So it seems, missed it while doing the review. Sorry!